.github/workflows/docker.yaml

```diff
@@ -50,6 +50,13 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       with:
         args: set image deployment.apps/celery-beat celery-beat=gcr.io/cdssnc/notify/api:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=/github/home/.kube/config
+    - name: Apply celery-sms
+      uses: docker://gcr.io/cdssnc/aws-kubectl:latest
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      with:
+        args: set image deployment.apps/celery-sms celery-sms=gcr.io/cdssnc/notify/api:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=/github/home/.kube/config
     - uses: cds-snc/notification-pr-bot@master
       env:
         TOKEN: ${{ secrets.TOKEN }}
```

Comment:
why do we need these docker changes?

Reply:
This ties into the PR here: https://github.com/cds-snc/notification-staging-tf/pull/4

It all ensures that the new command and new celery worker get run and deployed.

Reply:
I will need the other PR approved in order to test this as well :)